src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -100,26 +113,33 @@ protected AbstractBuilding(Colony colony, BlockPos pos)
     public abstract int getMaxBuildingLevel();
 
     /**
-     * Add build to a mapping.
-     * <code>buildingClass</code> needs to extend {@link AbstractBuilding}
-     * <code>parentBlock</code> needs to extend {@link AbstractBlockHut}
+     * Add build to a mapping. <code>buildingClass</code> needs to extend
+     * {@link AbstractBuilding} <code>parentBlock</code> needs to extend
+     * {@link AbstractBlockHut}
      *
-     * @param name              name of building.
-     * @param buildingClass     subclass of AbstractBuilding, located in {@link com.minecolonies.colony.buildings}
-     * @param parentBlock       subclass of Block, located in {@link com.minecolonies.blocks}
+     * @param name
+     *            name of building.
+     * @param buildingClass
+     *            subclass of AbstractBuilding, located in
+     *            {@link com.minecolonies.colony.buildings}
+     * @param parentBlock
+     *            subclass of Block, located in {@link com.minecolonies.blocks}
      */
-    private static void addMapping(String name, Class<? extends AbstractBuilding> buildingClass, Class<? extends AbstractBlockHut> parentBlock)
+    private static void addMapping(String name, Class<? extends AbstractBuilding> buildingClass,
+            Class<? extends AbstractBlockHut> parentBlock)
     {
         if (nameToClassMap.containsKey(name) || classNameHashToClassMap.containsKey(buildingClass.getName().hashCode()))
         {
-            throw new IllegalArgumentException("Duplicate type '" + name + "' when adding AbstractBuilding class mapping");
+            throw new IllegalArgumentException("Duplicate type '" + name
```

Comment:
You can look at the diff directly on Github. There you see everithat changed. 

Am 15.08.2016 9:53 vorm. schrieb Isfirs <notifications@github.com>:

In src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java:

> { > if (nameToClassMap.containsKey(name) || classNameHashToClassMap.containsKey(buildingClass.getName().hashCode())) > { > - throw new IllegalArgumentException("Duplicate type '" + name + "' when adding AbstractBuilding class mapping"); > + throw new IllegalArgumentException("Duplicate type '" + name 

It was a failure from my and my IDE. I prefer to perform format, organize imports and clean up on every save in my personal projects. I created a complete new workspace now and try to revert that. Can you make a list or tell me how to obtain a list of affected files?
I guess, I could do a git diff on my achievement HEAD and the develop HEAD and use the --file-name flag?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or mute the thread.